data/models/canteen.py

```diff
@@ -156,6 +156,10 @@ class EconomicModel(models.TextChoices):
         null=True, blank=True, verbose_name="participnte à l'expérimentation réservation"
     )
 
+    vegetarian_expe_participant = models.BooleanField(
```

Comment:
This field isn't included in the canteen serialiser so is never updated

Reply:
https://github.com/betagouv/ma-cantine/pull/1237/commits/340f71636b54b8c74afdf4911d7b5e341b55a6bf  :heavy_check_mark:  Good catch